File: src/main/java/com/fifo/ticketing/domain/performance/mapper/GradeMapper.java
package com.fifo.ticketing.domain.performance.mapper;

import com.fifo.ticketing.domain.performance.dto.GradeResponseDto;
import com.fifo.ticketing.domain.performance.entity.Grade;

public class GradeMapper {

    public static GradeResponseDto toDto(Grade grade) {
        return new GradeResponseDto(grade.getGrade(), grade.getSeatCount(),
                grade.getDefaultPrice());
    }
}

File: src/main/java/com/fifo/ticketing/domain/performance/mapper/GradeMapper.kt
package com.fifo.ticketing.domain.performance.mapper

import com.fifo.ticketing.domain.performance.dto.GradeResponseDto
import com.fifo.ticketing.domain.performance.entity.Grade

object GradeMapper {
    @JvmStatic
    fun toDto(grade: Grade) = GradeResponseDto(
        grade.grade, grade.seatCount,
        grade.defaultPrice
    )
}
